### ISSUE TITLE
Linux desktop app fails to run on TAILS and Whonix.

### ISSUE THREAD

**Issue by copecu** [Nault/Nault#301]
Hi,

version: Nault-1.9.7-Linux.AppImage

output:

`[2473:0214/091506.103104:FATAL:setuid_sandbox_host.cc(158)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /tmp/.mount_Nault-hdAgji/chrome-sandbox is owned by root and has mode 4755.
Trace/breakpoint trap (core dumped)`

**Comment by BitDesert** [comment]
Did you check that?
`You need to make sure that /tmp/.mount_Nault-hdAgji/chrome-sandbox is owned by root and has mode 4755`

**Comment by copecu** [comment]
> Did you check that?
> `You need to make sure that /tmp/.mount_Nault-hdAgji/chrome-sandbox is owned by root and has mode 4755`

Yes, but when I check /tmp, after the app fails, this directory did not exist.

I run another app with appimage "format" and it runs without problems.

Thanks for your comment.

**Comment by BitDesert** [comment]
Can you check if another app that uses electron works via AppImage? For example https://github.com/lensapp/lens

Could be related to https://github.com/electron/electron/issues/17972

**Comment by copecu** [comment]
> Did you check that?
> `You need to make sure that /tmp/.mount_Nault-hdAgji/chrome-sandbox is owned by root and has mode 4755`

Yes, but when I check /tmp, after the app fails, this directory did not exist.

I run another app with appimage "format" and it runs without problems.

Thanks for your comment.

> Can you check if another app that uses electron works via AppImage? For example https://github.com/lensapp/lens
> 
> Could be related to [electron/electron#17972](https://github.com/electron/electron/issues/17972)

Lens-4.0.8.AppImage fails in the same way, also TAILS and Whonix.

Greetings.
